src/extension.ts

```diff
@@ -3,12 +3,13 @@ import * as path from 'path'
 import * as replaceInFile from 'replace-in-file'
 import * as fs from 'fs'
 
-function createInput(): Thenable<string | undefined> {
+function createInput(filename): Thenable<string | undefined> {
```

Comment:
Please add a type declaration to `filename`. Other than that it looks good to me